src/main/java/jenkins/metrics/api/Metrics.java

```diff
@@ -331,9 +331,18 @@ public void stop() throws Exception {
 
     /**
      * provides the health check related metrics.
+     * 
+     * @deprecated use HealthCheckMetricsProvider
+     */
+    @Deprecated
```

Comment:
`@Restricted(NoExternalUse.class)` to force people to remove the old reference when they update the dependency